### ISSUE TITLE
Add pipeline for benchmarking SVs

### ISSUE THREAD

**Issue by rickymagner** [broadinstitute/palantir-workflows#128]
This issue serves as a discussion hub for development around a pipeline for benchmarking SVs. Development currently is happening on the [rm_benchmark_svs](https://github.com/broadinstitute/palantir-workflows/tree/rm_benchmark_svs) branch. There you can find the following useful documents:
- A [README](https://github.com/broadinstitute/palantir-workflows/tree/rm_benchmark_svs/BenchmarkSVs/README.md) which contains documentation on the current state of the pipeline and visualization tools included.
- A [Roadmap](https://github.com/broadinstitute/palantir-workflows/tree/rm_benchmark_svs/BenchmarkSVs/Roadmap.md) which contains checklists of items to do in the near future before the first major release.

If you have comments, issues using the pipeline, etc. please add them to this thread. I will post updates here during development as well.

**Comment by samuelklee** [comment]
Thanks again for the demo just now, @rickymagner! Things are looking great.

I've just shared the [kage-lite-dev-1](https://app.terra.bio/#workspaces/broad-firecloud-dsde/kage-lite-dev-1) Terra workspace with you. This currently hosts WDLs for running KAGE, PanGenie, and a leave-one-out evaluation that calculates weighted genotype concordance of imputed short-read genotypes with the panel genotypes for the left-out sample (stratified by length and genomic context, although I also have some offline versions that stratify by AF as well).

The workspace bucket contains some panel VCFs that may be interesting to throw through your prototype: 
- 32 HGSVC2 LR assemblies: `gs://fc-947cb896-7721-4f4d-8747-902d3a3394ee/20200825_pav-all-samples-freeze3-final.vcf.gz`
- 3202 SR 1kG + high coverage trios SR samples (chr1; in the bucket, you'll also see versions with some population annotations stripped out, subset to AF >= 0.001, symbolic alleles dropped, etc.): `gs://fc-947cb896-7721-4f4d-8747-902d3a3394ee/1kGP_high_coverage_Illumina.chr1.filtered.SNV_INDEL_SV_phased_panel.vcf.gz`

And now some runs that contain genotyped VCFs from KAGE and PanGenie. For these, you'll find VCFs in corresponding shards for each left-out sample:
- `call-KAGEPlusGLIMPSECase/../[SAMPLE_NAME].kage.glimpse.vcf.gz` for the result of KAGE genotyping + GLIMPSE imputation
- `call-KAGEPlusGLIMPSECase/../[SAMPLE_NAME].kage.vcf.gz` for the result of KAGE genotyping
- `call-call-PanGenieCase/../[SAMPLE_NAME]_genotyping.vcf.gz` for the result of PanGenie

You'll also find corresponding plots in the `call-CalculateMetrics*` folders (these are the 1D plots I've been showing in recent meetings).

Here are the runs:
- [11 LOO samples vs. the HGSVC2 LR panel over chr1+2](https://app.terra.bio/#workspaces/broad-firecloud-dsde/kage-lite-dev-1/job_history/763ec5ab-ef5c-4211-b9b1-1e041f9cf454)
- [1 LOO sample vs. the 1kG+trios SR panel over 10Mbp of chr1](https://app.terra.bio/#workspaces/broad-firecloud-dsde/kage-lite-dev-1/job_history/2af24018-421a-4a05-850d-065885bbe722)

As we discussed, it would be great to see the panels compared against each other, as well as the various combinations of panel vs. case. Might also be interesting to check each against your current gold standard set as well (although I may need to actually run that particular sample to generate the case VCFs!)

Finally, here's a (very rough first cut) of the sort of benchmarking heatmap that might be useful:
![KAGE+GLIMPSE](https://github.com/broadinstitute/palantir-workflows/assets/11076296/bec360c3-b85c-42ff-9c05-78e6c70753d9)
I was planning on adding some annotations to each cell, but hovering tooltips would be even better!
